package/kedro_viz/models/flowchart.py

```diff
@@ -407,18 +407,14 @@ def inputs(self) -> Set[str]:
         Intuitively, the set of inputs for this modular pipeline is the set of all
         external and internal inputs, excluding the ones also serving as outputs.
         """
-        return (self.external_inputs | self.internal_inputs) - (
-            self.external_outputs | self.internal_outputs
-        )
+        return self.external_inputs | (self.internal_inputs - self.internal_outputs)
```

Comment:
```suggestion
        return (self.external_inputs | self.internal_inputs) - self.internal_outputs
```

Nit. It's identical but it is slightly closer to what the docstring described.

> Intuitively, the set of inputs for this modular pipeline is the set of all
        external and internal inputs, excluding the ones also serving as outputs.